admin-console/src/main/resources/liquibase/admin-console/changeSetServ.xml

```diff
@@ -13,4 +13,8 @@
 
     <changeSet id="00000000000001_adminconsole_dataServ_restore" author="entando" context="restore" />
 
+    <changeSet id="20230118000001_adminconsole_dataServ_remove_" author="entando">
```

Comment:
@entgigi there is a naming convension for the id of changeset

Reply:
I changed it, and added a preconditioncheck just in case, please check it